karapace/utils.py

```diff
@@ -15,6 +16,7 @@
 import types
 
 log = logging.getLogger("KarapaceUtils")
+BLACKLIST_DURATION = 300
```

Comment:
Where is this used?

Reply:
nowhere.... artefact from alternate solution, will remove